Samples/SDKFeatures/SDKFeatures/ViewControllers/ActionPlans/MHVActionPlansListViewController.m

```diff
@@ -0,0 +1,187 @@
+//
+// MHVActionPlansListViewController.m
+// SDKFeatures
+//
+// Copyright (c) 2017 Microsoft Corporation. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import "MHVActionPlansListViewController.h"
+#import "MHVActionPlanDetailViewController.h"
+#import "MHVConnection.h"
+#import "MHVActionPlansApi.h"
+
+@interface MHVActionPlansListViewController ()
+
+@property (nonatomic, strong) NSMutableDictionary *actionPlans;
+@property (nonatomic, strong) MHVConnection *connection;
+
+@end
+
+@implementation MHVActionPlansListViewController
+
+- (id)initWithTypeClass:(Class)typeClass useMetric:(BOOL)metric
+{
+    self = [super init];
+    return self;
+}
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    _actionPlans = [[NSMutableDictionary alloc] init];
+    
+    [self.navigationController.navigationBar setTranslucent:FALSE];
+    self.navigationItem.title = @"Action Plans List";
+    
+    self.tableView.dataSource = self;
+    self.tableView.delegate = self;
```

Comment:
May fix in a later checkin, won't fix right now since it's following the existing pattern.